doc/news/changes/incompatibilities/20200326NiklasFehn

```diff
@@ -0,0 +1,7 @@
+Changed: The function ParameterHandler::parse_input() can now also parse xml- and json-files,
+depending on the file suffix. We treat implicitly all files not ending with .xml or
+.json as standard parameter files: however, in the case that the file does not end with .ptr a warning
```

Comment:
ptr? Here, I would formulate as if the function already contains if-statements for all current types, prm, xml, json, and everything else is a backward compatibility issue